dev_environment/setup_dev_env.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# this script need to run as root account, check it
+if [[ $EUID -ne 0 ]]; then
+    echo "This script must be run as root"
+    exit 1
+fi
+
+# Get cpu architecture, arm or amd
+ARCH=$(uname -p)
+if [[ "$ARCH" = "arm" ||  "$ARCH" = "aarch64" ]];
+then
+  echo "Setting l3af dev environment for arm"
+  arch=arm64
+elif [ "$ARCH" = "x86_64" ];
+then
+  echo "Setting l3af dev environment for amd64"
+  arch=amd64
+fi
```

Comment:
If the above two don't match this can cause issue.  On my mac for example it comes back as "i386".  If we can't match a supported platform arch we should exit here.

Reply:
I added the exit for un-supported cpu architecture.  Did your MAC return i386 for uname -p or arch command?  If it returned i386 for uname -p, can you retry the script?  I included i386 in the script.